lib/src/defines.dart

```diff
@@ -0,0 +1,7 @@
+import 'package:flutter/widgets.dart';
+
+const defaultTitlePadding = EdgeInsets.only(
```

Comment:
Was there any preferences as to how these values were chosen?

Reply:
Backward compatibility. These were lifted from your previous implementation.